### PR TITLE
Include section README section on the new CLA

### PR DESCRIPTION
**Pull request by schubert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -271,6 +271,11 @@ Known issues:
     Alternatively you can use the (beta) Native macOS docker client now available
     in docker 1.12.
 
+## Contributing
+
+If you have not previously done so, please fill out and
+submit the https://cla.pivotal.io/sign/greenplum[Contributor License Agreement].
```
Comment (danielgustafsson) [inline README.md]: The link should be formatted like below for Markdown:
```markdown
[Contributor License Agreement](https://cla.pivotal.io/sign/greenplum)
```

Comment (schubert) [inline README.md]: Oh jeez. 

I took this from the CLA tool documentation verbatim and didn't notice. I'll let them know have broken markdown in their docs. I'll fix. 

I really was just doing this to test the CLA tool integration.

Comment (danielgustafsson) [inline README.md]: That makes sense, the CLA tool is using AsciiDoc instead of Markdown and the above patch was in .adoc format.

**Comment by amilkh** [comment]
+1

**Comment by danielgustafsson** [comment]
+1

**Pull request merged**